### PR TITLE
Add CONTIBUTING.md.

### PR DESCRIPTION
**Pull request by ciechowoj**
I gathered some code style guidelines here. Tell me what do you think about this.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Code Style for Contributors
+- Use camelCase for functions and PascalCase for types.
+- Do not use tabs, indent code with 4 spaces.
+
+- Put a space between foreach, if, while etc. and the opening parenthesis.
+
+- Prefer to not include an empty set of parentheses in cases where there are only template parameters.
```
Comment (jacob-carlborg) [inline CONTRIBUTING.md]: Please use `*` for bullet point lists.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+```
```
Comment (jacob-carlborg) [inline CONTRIBUTING.md]: If you append a `d` to the end of the ``` characters it will add syntax highlighting.

**Comment by jacob-carlborg** [comment]
Please mention somewhere that a line should be preferred to be within 80 columns.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ciechowoj) [inline CONTRIBUTING.md]: Fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ciechowoj) [inline CONTRIBUTING.md]: Fixed.

**Comment by ciechowoj** [comment]
> Please mention somewhere that a line should be preferred to be within 80 columns.
Done.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Code Style for Contributors
+
+* Use camelCase for functions and PascalCase for types.
+* Do not use tabs, indent code with 4 spaces.
+* If possible, a line of code should be at most 80 characters long.
+* Put a space between foreach, if, while etc. and the opening parenthesis.
+
+* Prefer to not include an empty set of parentheses in cases where there are only template parameters.
```
Comment (jacob-carlborg) [inline CONTRIBUTING.md]: On the preview of this file, there are looks to be extra space in-between some of the bullet points. Try removing the extra newline (7) and see if that helps. Same thing further down.

Comment (ciechowoj) [inline CONTRIBUTING.md]: Fixed.

**Comment by jacob-carlborg** [comment]
👌

**Pull request merged**